scalafmt-tests/src/test/resources/scala3/MatchType.stat

```diff
@@ -52,15 +52,13 @@ type Concat[X <: Tuple, Y <: Tuple] <: Tuple = X match {
           x1 *: Concat[xs1, Y]
          }
 >>>
-type Concat[
-    X <: Tuple,
-    Y <: Tuple
-] <: Tuple = X match {
-  case Unit =>
-    Y
-  case x1 *: xs1 =>
-    x1 *: Concat[xs1, Y]
-}
+type Concat[X <: Tuple, Y <: Tuple] <:
+  Tuple = X match {
```

Comment:
would it break after `=` instead if we increase `maxColumn`?

Reply:
Checked and yes.